.buildkite/build-static-release.sh

```diff
@@ -35,7 +35,7 @@ sed -i.bak "s/0\\.0\\.0/${release_version}/" sorbet-static.gemspec
 if [[ "mac" == "$platform" ]]; then
     # Our binary should work on almost all OSes. The oldest v8 publishes is -14
     # so I'm going with that for now.
-    for i in {14..21}; do
+    for i in {14..22}; do
         sed -i.bak "s/Gem::Platform::CURRENT/'universal-darwin-$i'/" sorbet-static.gemspec
```

Comment:
Thoughts on just doing a single `universal-darwin` like grpc does? Unless there's some old Rubygems versions which don't support that, that seems like it would make this simpler.

Reply:
I have no plans to change this as it should work for the time being. If you would like to research and test an alternative be my guest!